packages/grid/x-data-grid-premium/src/hooks/features/clipboard/useGridClipboardImport.ts

```diff
@@ -208,10 +209,12 @@ function defaultPasteResolver({
   pastedData,
   apiRef,
   updateCell,
+  pagination,
 }: {
   pastedData: string[][];
   apiRef: React.MutableRefObject<GridApiPremium>;
   updateCell: CellValueUpdater['updateCell'];
+  pagination: boolean;
```

Comment:
To make it clear where it comes from
```suggestion
  pagination: DataGridPremiumProcessedProps['pagination'];
```